### DISCUSSION TITLE
very small number of peptides are found in alphaDIA if one uses empirical library from MS Fragger

### DISCUSSION THREAD

**Discussion by Yanidzilla** [MannLabs/alphadia#224] [General]
![Screenshot 2024-06-07 185254](https://github.com/MannLabs/alphadia/assets/147511279/cafbb301-10f6-4085-bed0-51ed8617bf16)
I am using the 1.6.2 windows version:
1) If I use the predict library option in alphaDIA, it gives "error No objects to concatenate", this was already mentioned in another issue;
2) If I input a .fasta together with a .tsv empirical library (that generated from MS Fragger and works very good with DIA-NN), then alphaDIA goes through but it gives me very little number of peptides compared with DIA-NN and Spectronaut (see enclosed screenshot).

So I wonder, it is the problem of my empirical library or alphaDIA doesn't go well with .tsv lib.
thx

**Comment by GeorgWa** [comment]
Hi, can you attach the log file?

**Comment by Yanidzilla** [comment]
[log.txt](https://github.com/user-attachments/files/15744543/log.txt)
Here is one log example, I repeat the same experiment in my Windows machine and my linux machine. This is the log from the linux machine. Cann't find the log for the windows anymore (probably it was covered by another log, I am keeping doing some tests).

Best regards :)

**Comment by GeorgWa** [comment]
Hi,

There's currently no optimization of parameters like MS2, RT, IM etc. in alphaDIA like you know it from DIANN or Spectronaut. This will be part of the next major release. 

In your case it looks like the RT tolerance is most likely too narrow based on the gradient and library. Maybe try setting the initial tolerance in the calibration tab to 500 and the final tolerance to 300. What gradient/method was this?

**Comment by GeorgWa** [comment]
We are trying to build a comprehensive benchmarking suite to make sure we get expected performance across methods and gradients.

**Comment by Yanidzilla** [comment]
Thanks Georg, It is a HYE dataset, the empirical library was generated by MSFragger.

**Comment by GeorgWa** [comment]
Okay, let me know if the mentioned settings solve it. I will move this to the discussions section 👍🏻

**Reply by Yanidzilla** [reply]
Thank Georg, I will test the mentioned settings probably in July (now is the summer vacation season :P). I will write you the feedbacks after testing.
